src/connections/cloudflare.ts

```diff
@@ -57,40 +62,15 @@ export class CloudflareD1Connection implements Connection {
      * @returns Promise<{ data: any, error: Error | null }>
      */
     async query(
-        query: string,
-        parameters?: Record<string, any>[]
-    ): Promise<{ data: any; error: Error | null }> {
+        query: Query
+    ): Promise<{ data: any; error: Error | null; query: string }> {
         if (!this.apiKey) throw new Error('Cloudflare API key is not set')
         if (!this.accountId) throw new Error('Cloudflare account ID is not set')
         if (!this.databaseId)
             throw new Error('Cloudflare database ID is not set')
         if (!query) throw new Error('A SQL query was not provided')
 
-        /**
-         * The Cloudflare API requires the query to be in the following format:
-         * SELECT * FROM table WHERE property = ?
-         *
-         * The parameters object should be an array of values that will replace
-         * the `?` in the query in the order they appear. We need to manipulate
-         * the query string to replace `:property` with `?` and extract the
-         * property names to be used as keys in the parameters object.
-         */
-        let queryParameters = query.match(/:[a-zA-Z0-9]+/g) ?? []
-        query = query.replace(/:[a-zA-Z0-9]+/g, '?')
-        let params = queryParameters
-
-        params.forEach((param, index) => {
-            let key = param.replace(':', '')
-
-            if (
-                parameters &&
-                parameters.length > 0 &&
-                parameters[0].hasOwnProperty(key)
-            ) {
-                params[index] = parameters[0][key]
-            }
-        })
-
+        const rawSQL = constructRawQuery(query)
```

Comment:
Might make more sense to move that down closer to where it's used. I thought this was used in the following fetch